SETUP.md

```diff
@@ -18,20 +18,22 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 1. `gem install bundler -v 1.17`
 1. `rbenv rehash`
 1. `cd code-dot-org`
-1. `bundle install` (Problems running this step? See [tips](#bundle-install-tips) below.)
+1. `bundle install`
+    * This step often fails to due environment-specific issues. Look in the [Bundle Install Tips](#bundle-install-tips) section below for steps to resolve many common issues.
 1. `bundle exec rake install:hooks`
     <details>
-        <summary>Troubleshoot: `rake aborted!..` </summary>
+      <summary>Troubleshoot: `rake aborted!..` </summary>
```

Comment:
"rake aborted" is likely to appear in almost any failure here. I would suggesting including at least `rake aborted! Gem::LoadError...`